### ISSUE TITLE
Update `@yamada-ui/slider` accessibility

### ISSUE THREAD

**Issue by nakayan5** [yamada-ui/yamada-ui#1727]
### Description

related https://github.com/yamada-ui/yamada-ui/issues/1505

This issue aims to address accessibility deficiencies in our slider component as per W3C guidelines.
The component currently does not meet the accessibility standards outlined in the W3C documentation.

The goal is to ensure the component is accessible to all users, including those using assistive technologies.
Ensure that all unchecked items in this checklist should be addressed.

- [ ] If the value of aria-valuenow is not user-friendly, e.g., the day of the week is represented by a number, the [aria-valuetext](https://w3c.github.io/aria/#aria-valuetext) property is set to a string that makes the slider value understandable, e.g., "Monday".

### Problem Statement/Justification

improve accessibility

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

**Comment by cidkumagai** [comment]
I'll try it.
